src/Extensions/Hosting.Services.Web/ApplicationBuilderExtensions.cs

```diff
@@ -23,23 +24,47 @@ public static class ApplicationBuilderExtensions
 		/// Adds the default exception handling logic that will display a developer exception page in develop and a short message during deployment
 		/// </summary>
 		/// <param name="builder">Application builder</param>
+		/// <param name="environment">Enviroment information</param>
 		/// <param name="enableCorrelationHeaderBackwardCompatibility">Set it to true if the service needs to accept a request with old style correlation</param>
-		public static IApplicationBuilder UseOmexMiddlewares(this IApplicationBuilder builder, bool enableCorrelationHeaderBackwardCompatibility)
+		public static IApplicationBuilder UseOmexMiddlewares(
```

Comment:
Middlewares strikes me as odd, Middleware would sound better without losing the meaning.

Reply:
I've wanted to emphasize there are many of them, not sure how grammatically correct it is.